pkg/template/core/starlark_struct.go

```diff
@@ -40,3 +45,50 @@ func (s *StarlarkStruct) AttrNames() []string {
 	})
 	return keys
 }
+
+func (s *StarlarkStruct) Get(key starlark.Value) (val starlark.Value, found bool, err error) {
+	if attr, ok := NewStarlarkValue(key).AsGoValue().(string); ok {
+		val, err = s.Attr(attr)
+		return val, val != nil, err
```

Comment:
seems like it would better to call s.data.Get for an explicit found boolean instead of checking on val!=nil which is "non-coded" contract from Attr() method.

Reply:
And, in fact, `Get()` should return `starlark.None`, not `nil` when no error occurred but the item was not found.